commands/acl.md

```
@@ -0,0 +1,135 @@
+
+    * count: (Optional) Limits the number of entries in the log.
+
+#### Example
+
+```
+> ACL LOG 10
+```
+
+## Notes
+
+    The ACL command is available only to users with administrative privileges.
```

Markdownlint (0.35.0)

Expected: fenced; Actual: indented
Code block style

(MD046, code-block-style)